.eslintrc

```diff
@@ -8,6 +8,37 @@
     "max-len": [2, 120],
     "react/jsx-filename-extension": 0,
     "import/no-extraneous-dependencies": 0,
-    "react/prefer-stateless-function": [2, { "ignorePureComponents": true }]
+    "react/prefer-stateless-function": [2, { "ignorePureComponents": true }],
+
+    // Addressed.
+
+    // We've decided we don't want this for now as it makes writing JSX more painful.
+    "function-paren-newline": 0,
```

Comment:
What value does the config have for this? I don't think fully disabling it is a good idea. The `minItems` options seems like what we want. Any examples of how it makes JSX painful?